metadata-ingestion/src/datahub/ingestion/source/tableau/tableau_common.py

```diff
@@ -910,6 +930,40 @@ def make_filter(filter_dict: dict) -> str:
     return filter
 
 
+def query_metadata_cursor_based_pagination(
```

Comment:
is this cursor stuff still relevant after your other graphql simplifications?

Reply:
Yup it is relevant as per tableau documentation.